packages/components/src/Brand/_docs/Brand.stories.tsx

```diff
@@ -25,42 +48,42 @@ DefaultStory.args = {
   reversed: false,
 }
 
-const StickerSheetTemplate: StoryFn<{ isReversed: boolean }> = ({
-  isReversed,
-}) => (
-  <StoryWrapper isReversed={isReversed}>
-    <StoryWrapper.Row rowTitle="Logo Horizontal">
-      <Brand
-        alt="Culture Amp"
-        variant="logo-horizontal"
-        reversed={isReversed}
-      />
-    </StoryWrapper.Row>
-    <StoryWrapper.Row rowTitle="Logo Vertical">
-      <Brand alt="Culture Amp" variant="logo-vertical" reversed={isReversed} />
-    </StoryWrapper.Row>
-    <StoryWrapper.Row rowTitle="Enso">
-      <Brand alt="Culture Amp" variant="enso" reversed={isReversed} />
-    </StoryWrapper.Row>
-    <StoryWrapper.Row rowTitle="Collective Intelligence">
-      <div style={{ width: "200px" }}>
-        <Brand
-          alt="Collective Intelligence"
-          variant="collective-intelligence"
-          reversed={isReversed}
-        />
-      </div>
-    </StoryWrapper.Row>
-  </StoryWrapper>
+/**
+ * `logo-horizontal`, `logo-vertical`, `enso`, `collective-intelligence`
+ */
+export const Variants: StoryFn = () => (
+  <StickerSheet>
+    <StickerSheet.Row rowTitle="Logo Horizontal">
```

Comment:
I think make it a single row if possible

Reply:
Can't seem to get the `'collective-intelligence'` variant to render this way... Even if I give the first three their own row collective intelligence disappears 🤔 
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/34709943/231620007-0f32f128-3d1c-4329-86c4-3e7737b3eb69.png">

Reply:
That's super weird... Maybe separate collective intelligence to it's own heading, cause it probably needs a little sentence for why it's used.